packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -274,3 +274,99 @@ class SampleAppCard extends React.Component<any, any> {
     });
   };
 }
+
+class SampleCard extends React.Component<any, any> {
```

Comment:
looks like there is not any reference to SampleCard.

Reply:
This is to avoid user from getting access to new card since the feature will not be release in next version